### PR TITLE
Added the contributors page and in navbar also i added link for contributors page.

### PR DESCRIPTION
**Pull request by tarunkumar2005**
Added the contributors page and in navbar also i added link for contributors page.

closes issue : #88 

https://github.com/user-attachments/assets/f8deaf96-bc13-4a58-9fb7-c71f03461afc

@ratnesh-maurya I have done it the way you said, check this PR and merge it if you have any issue you can tell me i will update it and assign this to me and put relevant labels like gssoc 2024 ext and hacktoberfest. 
i think it can be level 3 because the page is very important in open source to showcase the efforts of the people contributing in building it and it is not static it is dynamic as a new contributor is added to your repo it will be automatically added there so this is the beauty of this.

### CODE REVIEW

**Comment by tarunkumar2005** [comment]
Awersome repo properly and well structured and properly coded but not beginer friendly it took me 10 min to understand the structure of the project then where which file is there and how to add things and so but i figured it out and amazing experience and got to learn new things from this. Thank you for assigning this issue to me, i got to work on a properly structured and large codebase (according to me atleast i haven't worked on this much big codebase before.)

**Comment by ratnesh-maurya** [comment]
Awesome @tarunkumar2005  there is small change **remove contributor tab** from the sidebar and  github icon in the header and on clicking that icon contributors page should show up .

File: Frontend/src/configs/navigation.config/index.ts [hunk 1]
```diff
@@ -86,5 +86,15 @@ const navigationConfig: NavigationTree[] = [
         authority: ['SUPERADMIN'],
         subMenu: [],
     },
+    {
```
Comment (ratnesh-maurya) [inline Frontend/src/configs/navigation.config/index.ts:89]: remove this from the sidebar and add it to the header

**Review by ratnesh-maurya — CHANGES_REQUESTED** [review]

**Comment by ratnesh-maurya** [comment]
Perfect ✌🏻

**Review by ratnesh-maurya — APPROVED** [review]

**Pull request merged**